### ISSUE TITLE
Set Up Bank Vault Consumer Job

### ISSUE THREAD

**Issue by alexkeating** [HausDAO/daohaus-monorepo#195]
## Overview

Once our Kubernetes Cluster is set up we can create jobs that run. This job will be used in our Hub App for DAO processing in the DAOs Table (specifically for the vault data).

This depends on having our own Ceramic Node deployed (#173 will need to be completed first) so it is **blocked** until our node is deployed.

This also will need a Ceramic schema for the fields (we can break this into a separate issue if we need).

These are the types  from `dao-data`:
```
export interface DaoTokenBalances {
  safeAddress: string;
  fiatTotal: number;
  tokenBalances: TokenBalance[];
}
```
```
export type TokenBalance = {
  token: TokenInfo | null;
  tokenAddress: string | null;
  balance: string;
  ethValue: string;
  timestamp: string;
  fiatBalance: string;
  fiatConversion: string;
  fiatCode: string;
};
```

## Details

- **Schema (fields needed)**:
  -  Look to the type definitions used in `dao-data` to inform what we need in our schema
- Create a job that gets the vault data from DAOs in the processing queue and stores their vault data to Ceramic so that it can be accessed by the DAOs Table in Hub
- **Steps (`dao-data` package has functions for this )**:
  - Take each DAO off the queue (#194)
  - Query the Gnosis API by network and safe address
  - Create/update a DID for each DAO/safe with the token/vault info 
  - Does this then remove the DAO from the queue and then re-add every 5 minutes to keep the data fresh with any updates? 
 
**Questions/Future Tickets Considerations:**

- Update the `dao-data` SDK to query Ceramic instead of the Gnosis API directly
- In the future we may need a way to trigger this on demand (such as if a DAO sends a token immediately we'd need an updated token balance instead of one that may be have info that is ~5 minutes outdated)
  - For single time on demand hits the frontend app could leverage the `dao-data`  

## Deliverables

- DAO Bank Vault Consumer Job added to our infrastructure
- Potential Playbook documentation?

## Related Issues

- #195 
- #172 
- #196

**Comment by earth2travis** [comment]
Requires Helm

**Comment by earth2travis** [comment]
@alexkeating added https://github.com/HausDAO/daohaus-monorepo/issues/504
